tests/test_rigid_physics.py

```diff
@@ -430,3 +431,20 @@ def test_nonconvex_collision(show_viewer):
 
     if show_viewer:
         scene.viewer.stop()
+
+
+@pytest.mark.parametrize("xml_path", ["xml/test_equality.xml"])
+@pytest.mark.parametrize("gs_solver", [gs.constraint_solver.CG])
+@pytest.mark.parametrize("gs_integrator", [gs.integrator.Euler])
+@pytest.mark.parametrize("backend", [gs.cpu], indirect=True)
+def test_equality_joint(gs_sim, mj_sim):
+    # there is an equality constraint
+    np.testing.assert_allclose(gs_sim.rigid_solver.n_equalities, 1, 0.1)
```

Comment:
`assert gs_sim.rigid_solver.n_equalities == 1` would do fine in this case, or `assert (gs_sim.rigid_solver.n_equalities == 1).all()` in case it is a vector.